### PR TITLE
Lh edits

### PR DESCRIPTION
**Pull request by Farnazmdi**
Changed the colors this way to accommodate colorblind:
image

### CODE REVIEW

**Review by aarmey — CHANGES_REQUESTED** [review]
Please check your changes using Grammarly first.

**Comment by Farnazmdi** [comment]
Sorry about the typos..

File: manuscript/03.results.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ More specifically, one of the benchmarking studies we performed was with data ma
...
 
-![**Model performance versus the Wasserstein distance between states.** (a) Cartoon of how two states can vary in their phenotypic similarity. On the left, cells might be virtually indistinguishable (here based on shape). On the right, they might be so different that looking at one cell is sufficient to identify its state. (b) The distribution of G1 duration is varied in state 1 (blue) while the other state is kept constant. (c) State assignment accuracy versus the Wasserstein distance between state phenotypes. Each point represents the accuracy of state assignment for a lineage created by a set of parameters that yield the shown Wasserstein distance between the two state distributions. Either a K-means model was used with the phenotypes of each individual cell (orange) or the full tHMM model was applied (blue). The solid lines show a Lowess trendline of the model accuracy.](figure6.svg){#fig:wass}
```
Comment (aarmey) [inline manuscript/03.results.md:49]: It seems like the only fix here is replacing left/right with top/bottom. The new text is wrong; difference/similarity is judged between states, not within them.

File: manuscript/03.results.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ To correct for this effect in our model, we marked cells that encountered the st
...
+To evaluate how accurately a tHMM model could infer the behavior of multi-state cell populations, we used synthetic populations of cells in a wide variety of configurations, such as various populations sizes, abundance of the present states, etc. In each case, we determined that our implementation of a tHMM model could accurately infer the hidden states and parameters of a population given at least 100 cells. This synthetic data included situations that were both not censored (Figs. @fig:uncenSingle, @fig:uncenMulti, @fig:performUncenSingle, @fig:performUncenMulti) and censored due to cell death and experimental duration to be representative of experimental lineages (Figs. @fig:performSyn, @fig:performCenMulti, @fig:cenMulti). In addition to varying the number of cells in a population, we benchmarked populations with varied cell state percentages (Figs. @fig:prop4, @fig:real_5) and varied degrees of phenotypic differences (Figs. @fig:wass1, @fig:wass2, @fig:wass). This benchmarking consistently showed that the tHMM model would provide accurate results across a range of circumstances.
```
Comment (aarmey) [inline manuscript/03.results.md:37]: Remove "etc"—it's inexact. You can just say "such as X or Y."

File: manuscript/03.results.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 <!-- ASM proofed. -->
 
-To illustrate the unique value of lineage measurements in analyzing intra-tumoral and drug response heterogeneity, we plotted two sets of sample lineages from measurements of the breast cancer cell line AU565 (Fig. @fig:lineage a-b). The single cell lineages reveal a striking difference in cell cycle phase durations and cell division dynamics between the two samples. Population-level measurements, on the other hand, would be unable to identify this difference as the starting and ending cell numbers are not different. Measurements that record the history of cells (e.g., CFSE staining, Luria-Delbruck experiment) can help to identify these variations within cell populations but must make assumptions about the dynamics of heterogeneity [@DOI:10.1073/pnas.1715639115; @PMID:28607484]. Lineage measurements, by contrast, provide sufficiently rich temporal information to quantify the specific structure of the phenotypic heterogeneity.
```
Comment (aarmey) [inline manuscript/03.results.md:7]: What was the comment this aims to address? It's probably good in its present form—just want to check.

**Review by aarmey — CHANGES_REQUESTED** [review]

File: manuscript/03.results.md (hunk 3 above)
Comment (Farnazmdi) [reply manuscript/03.results.md:7]: "A 1-2 sentence description of the dataset at the start of this paragraph would be helpful. and perhaps include a sentence on cell cycle phases. Additionally, this assumes the reader knows what a cell lineage is, which might not be true for a comp reader"

**Comment by aarmey** [comment]
Great. Thanks!

**Pull request merged**